lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -59,7 +60,15 @@ def initialize(sandbox, aggregate_targets, pod_targets, analysis_result, install
         end
 
         def generate!
-          project = prepare
+          project_path = sandbox.project_path
+          build_configurations = @analysis_result.all_user_build_configurations
```

Comment:
you dont have to use the ivars here at all. You can remove the `@` for most of these